content/build/building/variables.md

```diff
@@ -426,6 +431,53 @@ Example:
 }
 ```
 
+### BUILDX_BAKE_GIT_AUTH_HEADER
+
+{{< introduced buildx 0.14.0 >}}
+
+Sets the HTTP authentication scheme when using a remote Bake definition in a private Git repository.
+This is equivalent to the [`GIT_AUTH_HEADER` secret](./secrets#http-authentication-scheme),
+but facilitates the pre-flight authentication in Bake when loading the remote Bake file.
+Supported values are `bearer` (default) and `basic`.
+
+Usage:
+
+```console
+$ export BUILDX_BAKE_GIT_AUTH_HEADER=basic
+```
+
+### BUILDX_BAKE_GIT_AUTH_TOKEN
+
+{{< introduced buildx 0.14.0 >}}
+
+Sets the HTTP authentication token when using a remote Bake definition in a private Git repository.
+This is equivalent to the [`GIT_AUTH_TOKEN` secret](./secrets#git-authentication-for-remote-contexts),
+but facilitates the pre-flight authentication in Bake when loading the remote Bake file.
+
+Usage:
+
+```console
+$ export BUILDX_BAKE_GIT_AUTH_HEADER=$(cat git-token.txt)
```

Comment:
```suggestion
$ export BUILDX_BAKE_GIT_AUTH_TOKEN=$(cat git-token.txt)
```

Reply:
Fixed this and also called out the private repo auth in the remote definitions page.